### PR TITLE
BAEL-4443 Reading HTTP ResponseBody as a String

### PR DESCRIPTION
**Pull request by oussama-benmahmoud**

### CODE REVIEW

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
Use a more specific package name, and remove the .gitignore

File: httpclient-2/src/test/java/com/baeldung/httpclient/ApacheHttpClientUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.baeldung.httpclient;
```
[the remark this answers is not in the document]
Comment (oussama-benmahmoud) [reply httpclient-2/src/test/java/com/baeldung/httpclient/ApacheHttpClientUnitTest.java]: Ok, I'll do this.

File: httpclient-2/.gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+*.class
```
[the remark this answers is not in the document]
Comment (oussama-benmahmoud) [reply httpclient-2/.gitignore]: I followed the httpclient module but ok I'll delete it.

**Review by KevinGilmore — APPROVED** [review]

**Comment by davidmartinezbarua** [comment]
retest this please

**Comment by lor6** [comment]
retest this please

**Comment by oussama-benmahmoud** [comment]
@lor6 the issue is related to a failed test in another module, where there's no link/dep with httpclient-2 module, I tried to run the test on my pc and it worked fine, can you try to rerun the pipeline please?

**Comment by davidmartinezbarua** [comment]
retest this please

**Pull request merged**